library/build.gradle

```diff
@@ -1,8 +1,11 @@
 apply plugin: "com.android.library"
+apply plugin: "com.jfrog.bintray"
+apply plugin: "com.github.dcendents.android-maven"
 
 apply from: "${rootDir}/gradle/android-common.gradle"
 
-group = "com.okta"
+group = "com.okta.android"
+version = "0.1.0"
```

Comment:
you don't need to redefine this. It's already in `"${rootProject.versionName}"` via the `ext` tag